src/components/nodes-table/nodes-table.js

```diff
@@ -47,10 +48,17 @@ export default {
 					label: 'Power',
 					customValue: (node) => this.getBatteryLevel(node),
 					customFormat: (value) => (value ? value + '%' : 'mains'),
-					customInfo: (node) =>
-						node.batteryLevel
-							? 'Battery level: ' + node.batteryLevel + '%'
-							: 'mains-powered',
+					customInfo: (node) => {
+						if (node.powerSource === 'mains') return 'mains-powered'
+						let levelInfo = ''
+						for (const ep in node.batteryLevels) {
+							if (levelInfo !== '') {
+								levelInfo += '/'
+							}
+							levelInfo += `${node.batteryLevels[ep]}%`
+						}
+						return `Battery level(s): ${levelInfo}`
+					},
```

Comment:
Maybe providing a `customDisplay` function instead of separate `customFormat` and `customInfo` would be better.
The function would then expect an object like the one provided by `getPowerInfo()`.
This way any table column could be customized in a better way (e.g. to show icons instead of 'Yes'/'No').

Reply:
Agree 👍🏼